trollmoves/server.py

```diff
@@ -67,15 +67,17 @@ class ConfigError(Exception):
 
 class Deleter(Thread):
 
-    def __init__(self):
+    def __init__(self, attrs):
         Thread.__init__(self)
         self.queue = Queue()
         self.timer = None
         self.loop = True
+        self._attrs = attrs
 
     def add(self, filename):
         LOGGER.debug('Scheduling %s for removal', filename)
-        self.queue.put((filename, time.time() + 30))
+        remove_delay = int(self._attrs.get('remove_delay', 1))
+        self.queue.put((filename, time.time() + 30 * remove_delay))
```

Comment:
Would it be better to have the delay directly in the config file without the hidden 30x multiplier?
```python
remove_delay = int(self._attrs.get('remove_delay', 30))
self.queue.put((filename, time.time() + remove_delay)
```

Reply:
I thought since the original is 30 (seconds that is) that should be default.

But we can use:
```
remove_delay = int(self._attrs.get('remove_delay', 30))
self.queue.put((filename, time.time() + remove_delay))
```

Reply:
Exactly :-) In the current version in PR the config value would multiply the default delay (30 s) and wouldn't be very intuitive thing to configure.

Reply:
ok